src/main/resources/carpet-extra.mixins.json

```diff
@@ -3,73 +3,71 @@
   "package": "carpetextra.mixins",
   "compatibilityLevel": "JAVA_8",
   "mixins": [
-    "DispenserBlockMixin",
```

Comment:
Please don't sort mixins unnecessarily

Reply:
Oops didn't notice the plugin was doing that. I'll fix it in a bit